flink-end-to-end-tests/flink-end-to-end-tests-pulsar/src/test/java/org/apache/flink/tests/util/pulsar/common/FlinkContainerWithPulsarEnvironment.java

```diff
@@ -43,12 +44,10 @@ private static String resourcePath(String jarName) {
         return TestUtils.getResource(jarName).toAbsolutePath().toString();
     }
 
-    @Override
-    protected Configuration flinkConfiguration() {
-        Configuration configuration = super.flinkConfiguration();
+    protected static Configuration getClusterConfiguration() {
+        Configuration configuration = new Configuration();
         // Increase the off heap memory for avoiding direct buffer memory error on Pulsar e2e tests.
         configuration.set(TASK_OFF_HEAP_MEMORY, MemorySize.ofMebiBytes(100));
```

Comment:
So we don't need the old flink configuration from its parent class, right?

Reply:
Actually we can remove this class. I think the initial purpose of this is for passing additional Flink configuration to FlnkContainersTestEnvironment. I added a constructor for supporting this.